lib/player.js

```diff
@@ -1868,12 +1868,24 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     // Pick the initial streams to play.
```

Comment:
I believe you should move this into the conditional as well.  There's no reason to choose a variant if one has already been chosen by the application, and debug logs from chooseVariant_() might be confusing if the result of that function is ignored.